frontend/vite.config.ts

```diff
@@ -3,6 +3,7 @@ import react from '@vitejs/plugin-react';
 import { defineConfig } from 'vite';
 import { nodePolyfills } from 'vite-plugin-node-polyfills';
 import svgr from 'vite-plugin-svgr';
+import { coverageConfigDefaults } from 'vitest/config';
```

Comment:
This would mean vitest and friends would need to move from devDependencies to dependencies? Because now they are required for `npm run build`, and `npm ci` doesn't install devDependencies.

I think that's the cause of the container build failure anyway.

Reply:
yeah but it's not really an issue because headlamp is not an npm package

Reply:
Not sure what that means, but what I'm trying to say is...

We only use `npm ci` for building the container image, that's why it's not installing the devDependencies and erroring in that CI check.

Reply:
oh sorry, I meant that I agree that vitest needs to move to normal dependencies and the fact that vitest will be moved is not a big issue since we're using devDeps mostly to speedup CI